dist/api_data.py

```diff
@@ -814,6 +814,15 @@ def __ge__(self, other):
                 directory must already exist. If the value is not an absolute path, the path
                 is relative to the database home (see @ref absolute_path for more information)'''),
         ]),
+    Config('rollback_to_stable', '', r'''
+        rollback tables to an earlier point in time, discarding all updates to checkpoint durable
+        tables that have commit times more recent than the current global stable timestamp''',
+        type='category', subconfig=[
+            Config('threads', '', r'''
```

Comment:
I prefer to go with the default value of 4 worker threads. It can be changed with WiredTiger configuration if the user wants.